mmv1/products/healthcare/FhirStore.yaml

```diff
@@ -263,6 +263,28 @@ properties:
                     resource is a recursive structure; when the depth is 2, the CodeSystem table will have a column called
                     concept.concept but not concept.concept.concept. If not specified or set to 0, the server will use the default
                     value 2. The maximum depth allowed is 5.
+                - !ruby/object:Api::Type::NestedObject
+                  name: lastUpdatedPartitionConfig
+                  description: |
+                    The configuration for exported BigQuery tables to be partitioned by FHIR resource's last updated time column.
+                  properties:
+                    - !ruby/object:Api::Type::Enum
+                      name: type
+                      required: true
+                      description: |
+                        Type of partitioning.
+                      default_value: :PARTITION_TYPE_UNSPECIFIED
```

Comment:
its failing because default and required and required are incompatible . We'll need to remove the default here.

Reply:
Thanks for pointing this out!